src/agent/state.js

```diff
@@ -286,15 +288,16 @@ StateResolver.prototype.isPathInNodeModulesDirectory_ = function(path) {
   return path.indexOf('node_modules') === 0;
 };
 
-StateResolver.prototype.resolveFrame_ = function(frame, resolveVars) {
-  var noArgs = [{
+StateResolver.prototype.resolveFrame_ = function(frame, resolveVars, currentFrame, totalFrames) {
+  var framesExceeded = (currentFrame >= totalFrames);
```

Comment:
Looking at all the code-paths through here, I believe `framesExceeded` will always be equal to `!resolveVars`. Am I reading things correctly?